### PR TITLE
 Functionality to allow admins to switch on/off the line endings conversion on import

### PR DESCRIPTION
**Pull request by zipho**
Functionality to allow admins to switch off the line endings conversion on import. 

This is a cleanup to PR https://github.com/galaxyproject/galaxy/pull/2803

### CODE REVIEW

File: tools/data_source/upload.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
 import sys
 import tempfile
 import zipfile
+import cProfile
```
Comment (jmchilton) [inline tools/data_source/upload.py]: Can you remove these changes?

Comment (zipho) [reply tools/data_source/upload.py]: These are now removed.

File: tools/data_source/upload.py [hunk 2]
```diff
@@ -12,7 +12,6 @@
 import sys
 import tempfile
 import zipfile
-import cProfile
```
Comment (jmchilton) [inline tools/data_source/upload.py]: But the other changes to this file seem to remain - the part that used the cProfile import.

Comment (zipho) [reply tools/data_source/upload.py]: I've removed the changes. The initial commit did'nt carry these.

**Comment by zipho** [comment]
Is there any reasons this has not been merged. I am happy to comply with any suggestions.

**Comment by nsoranzo** [comment]
@galaxybot test this

**Comment by nsoranzo** [comment]
@galaxybot test this

**Comment by jmchilton** [comment]
@galaxybot test this

**Comment by nsoranzo** [comment]
@galaxybot test this

**Comment by nsoranzo** [comment]
LGTM, but ping @martenson (the libraries guru).

**Comment by martenson** [comment]
works well, thanks @zipho for the contribution and @jmchilton with @nsoranzo for the review

**Pull request merged**